exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/SqlConverter.java

```diff
@@ -187,12 +187,11 @@ public SqlNode parse(String sql) {
 
   public SqlNode validate(final SqlNode parsedNode) {
     try {
-      SqlNode validatedNode = validator.validate(parsedNode);
-      return validatedNode;
+      return validator.validate(parsedNode);
     } catch (RuntimeException e) {
       UserException.Builder builder = UserException
           .validationError(e)
-          .addContext("SQL Query", sql);
+          .addContext("SQL Query", (String) null);
```

Comment:
Why do we need to use it? `addContext()` with one parameter exists

Reply:
Removed `addContext` since it does not make sense in this case.